lib/Doctrine/DBAL/Platforms/DB2Platform.php

```diff
@@ -269,13 +269,8 @@ public function getTruncateTableSQL($tableName, $cascade = false)
 
     /**
      * This code fragment is originally from the Zend_Db_Adapter_Db2 class, but has been edited.
-     *
-     * @param string $table
-     * @param string $database
-     *
-     * @return string
      */
-    public function getListTableColumnsSQL($table, $database = null)
+    public function getListTableColumnsSQL(string $table, ?string $database = null) : string
```

Comment:
The nullability of the `$database` argument in `AbstractPlatform` methods is not final. It just reflects the current type. The type can be restricted to a non-nullable `string` in a subsequent PR.